### PR TITLE
Fix flaky add_user smoke test

### PR DESCRIPTION
**Pull request by kkorchak**
The main source of add_user being flaky is the behavior of 'Share Invite Link' modal on settings/identities/users. 
Sometimes it takes some time to present actual token id on UI while token url is already present. 
So Cypress verifies that URL present and goes forward causing it to copy corrupted URL of token
image
This causes test to fail later.
This is how UI looks when this happens:
image
The solution for this would be to wait for 1000ms after 'Share Invite Link' is open to make sure that full invite link is present.
image

Linear ticket: https://linear.app/acryl-data/issue/OBS-47/qa-fix-add-user-flaky-test
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ describe("add_user", () => {
         cy.visit("/settings/identities/users");
         cy.waitTextVisible("Invite Users");
 
-        cy.clickOptionWithText("Invite Users")
+        cy.clickOptionWithText("Invite Users").wait(1000);
```
Comment (jjoyce0510) [inline smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js:23]: Any chance we should wait until the URL matches a particular REGEX?

Or is it re-generating a token so we cannot use a regex match here?

Comment (kkorchak) [reply smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js:23]: yeah, it re-generating a token every time, maybe we can check for length of token in REGEX...

Comment (kkorchak) [reply smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js:23]: Tested this REGEX locally and it works. 
"/signup\?invite_token=\w{32}/"

File: smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+        cy.login();
 
         cy.visit("/settings/identities/users");
         cy.waitTextVisible("Invite Users");
 
-        cy.clickOptionWithText("Invite Users")
+        cy.clickOptionWithText("Invite Users");
 
-        cy.waitTextVisible(/signup\?invite_token=\w+/).then(($elem) => {
+        cy.waitTextVisible(/signup\?invite_token=\w{32}/).then(($elem) => {
```
Comment (jjoyce0510) [inline smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js:25]: nice!! thanks for adding the regex matching. this is much better

**Review by jjoyce0510 — APPROVED** [review]

**Review by jjoyce0510 — APPROVED** [review]

**Pull request merged**